### PR TITLE
effectiveOdds toFractional returns 'decimal/1' for 2+ selections

### PR DESCRIPTION
**Pull request by nulty**
- The industry standard for calculating effective odds when multiplying
   fractions seems to be that if the calculated fraction results in the
   denominator being greater than 1, the fraction is converted to decimal
   and placed over 1 for simplicity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,5 +110,18 @@ effectiveOdds(
...
+### Build
+
+`> rollup -c rollup.config.js`
+
+### Minify
+
+`> npm install -g uglify-es`
+
+`> uglifyjs dist/bet-return.js -o dist/bet-return.min.js`
+
```
Comment (revdan) [inline README.md:127]: I'm not sure these two steps are instrumental to contributing, as they would only need to be run when we're building a release (and would happen as part of the build script). The `test` and `test:cover` ones might be useful though.

**Review by revdan — APPROVED** [review]
Looks good 👍

File: README.md [hunk 2]
```diff
@@ -90,6 +90,18 @@ multiple({
 ```
 
 ### Effective Odds
+#### One selection
+
+```
```
Comment (hopeshigh) [inline README.md]: You are able to add a label of javascript for nice highlighting in Markdown. 
This line will then become:
"```javascript"

**Review by hopeshigh — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nulty) [reply README.md:127]: Good idea on the build script. I'm not going to create a build script here and now though, and I think it's worth leaving these instructions here until that build script is added.

**Pull request merged**